Description: Define QCA_EXPORT as Q_DECL_EXPORT on non-Windows systems so we
 can build with -fvisibility=hidden.
Author: Felix Geyer <debfx-pkg@fobos.de>

--- a/include/QtCrypto/qca_export.h
+++ b/include/QtCrypto/qca_export.h
@@ -44,6 +44,8 @@
 #define QCA_EXPORT Q_DECL_IMPORT
 #endif
 #endif
+#else
+#define QCA_EXPORT Q_DECL_EXPORT
 #endif
 #ifndef QCA_EXPORT
 #define QCA_EXPORT
